lib/Adapter/Git/GitFilesystem.php

```diff
@@ -23,7 +23,7 @@ public function __construct(FilePath $path)
 
     public function fileList(): FileList
     {
-        $gitFiles = $this->exec('ls-files');
+        $gitFiles = array_merge($this->exec('ls-files'), $this->exec('ls-files --others --exclude-standard'));
```

Comment:
I think this is equivalent to `$this->exec('ls-files --cached  --others --exclude-standard')

Reply:
Yes indeed, updating.